### ISSUE TITLE
negative argument (string repeat) in obfuscation for no-argument methods

### ISSUE THREAD

**Issue by onyxraven** [evanphx/newrelic-redis#21]
This may be specific to running fakeredis, but I was seeing 

```
ArgumentError: negative argument
from .../newrelic-redis-2.0.0/lib/newrelic_redis/instrumentation.rb:75:in `*'
```

This was happening when doing a multi/exec block, because the args list on those commands is empty.

**Comment by onyxraven** [comment]
Hm, sorry, I didn't include my details and I was going to do a PR, but ran out of time.

The args list I was seeing this error on:

Commands:
```
MULTI
LRANGE test 0 8
LTRIM test 0 8
EXEC
```

produces:
```
args = [[], ["test", 0, 8], ["test", 0, 8], []]
```

So its not the whole args list, but the list per command.

**Comment by evanphx** [comment]
Oh whoops. Ok! Fixed now and released 2.0.2.

**Comment by onyxraven** [comment]
you rock, thanks!
